### DISCUSSION TITLE
Different browsers options to open links with?

### DISCUSSION THREAD

**Discussion by pokegamer5547** [Docile-Alligator/Infinity-For-Reddit#929] [Ideas]
I don't want to open links in reddit using Chrome. I know I can disable chrome app but I think If would be great if there's an option to open custom tabs with using different browsers like Firefox, etc.

**Comment by Skyluker4** [comment]
I know you can change your default browser, but that's about it right now. I have my settings to open links in app, which really just embeds your default browser in the app. I'd also like to have the ability to have links open in other apps if they are not just web links. Like YouTube opening in the YouTube app and now just the custom brower tabs.
